jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -21,6 +21,8 @@
 import com.google.cloud.tools.jib.async.NonBlockingSteps;
 import com.google.cloud.tools.jib.blob.Blobs;
 import com.google.cloud.tools.jib.builder.BuildConfiguration;
+import com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.Result;
```

Comment:
Is this import necessary?

Reply:
Yea, it is actually necessary for the class to use `Result` as the template.